app/config/AWS.scala

```diff
@@ -78,7 +78,13 @@ class AWS {
     val request = new DescribeAutoScalingGroupsRequest().withAutoScalingGroupNames(autoscalingGroupName)
     val response = asgClient.describeAutoScalingGroups(request)
 
-    val group = response.getAutoScalingGroups.asScala.headOption
-    group.map(_.getTags.asScala.map { t => t.getKey -> t.getValue }(scala.collection.breakOut))
+    val maybeGroup = response.getAutoScalingGroups.asScala.headOption
+    maybeGroup.map {
```

Comment:
This change is necessary as `scala.collection.breakOut` is no longer available.

Reply:
I think the current use of the view function is redundant. I suspect we could either remove it, or place it before the map call. I'm not especially familiar with `view`s happy to be corrected if I've missed something.

Reply:
I've just had a play, and it looks like we'll need a view (or some other mechanism I don't yet know of) to go from a `mutable.Buffer` (the output of `.asScala`) to the output `Map` – doesn't really matter whether the `.view` happens before or after `.map`. Be happy if you know of a  better way of doing this, mind!

Reply:
Removing it and keeping the `to` seems to work fine for me. My understanding is that a `view` acts an optimisation when chaining multiple functions on a collection, which avoids intermediary representations. This would seem to be unnecessary in this case.

Reply:
Oh, nice, yeah that seems to work great. I've also replaced `to(Map)` with `toMap`, for no reason other than it looks a bit more official and less intimidating!

Reply:
Sounds good! Sorry, I know it's a minor thing, just interested to improve my understanding of views if I've got it wrong.